snapshots/v2.3.0/configuring-ember/handling-deprecations/index.html

```diff
@@ -140,7 +141,7 @@ <h3 class='anchorable-toc' id='toc_1-gather-deprecations-into-one-source'>1. Gat
 <p>The ember-cli-deprecation-workflow addon provides a command that will collect deprecations from your console and generate JavaScript code listing
 its findings.</p>
 
-<p>To collect deprecations, first run your in-browser test suite by starting your development server and navigating to <a href="http://localhost:4200/tests"><code>http://localhost:4200/tests</code></a>.  If your test suite isn&#39;t fully covering your app&#39;s functionality, you may also
+<p>To collect deprecations, first run your in-browser test suite by starting your development server and navigating to <code>http://localhost:4200/tests</code>.  If your test suite isn&#39;t fully covering your app&#39;s functionality, you may also
```

Comment:
We never properly copied our v2.6+ branch back to the `v2.3.0`, `v2.4.0` and `v2.5.0` branches of the guides, so when rebuilding those versions, we lost a few commits.